cmake/Modules/Findsystemd.cmake

```diff
@@ -30,5 +30,4 @@ set(Libsystemd_PROCESS_INCLUDES Libsystemd_INCLUDE_DIR)
 set(Libsystemd_PROCESS_LIBS Libsystemd_LIBRARY)
 
 # silence warnings
-set(Libsystemd_FIND_QUIETLY 1)
```

Comment:
Can you extract this change to a separate commit?

Reply:
Fixed